src/main/java/mapper/MappingInfoRepository.java

```diff
@@ -0,0 +1,54 @@
+package mapper;
+
+import annotation.MethodType;
+import controller.Controller;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import request.HttpRequest;
+
+public class MappingInfoRepository {
+
+    private static Map<String, Class<?>> exceptionMap = new HashMap<>();
```

Comment:
요렇게 쓸 수 있지 않나요? 저도 가물가물하네요...
```suggestion
    private static Map<String, Class<? extends Exception>> exceptionMap = new HashMap<>();
```